src/chap_2_functional.res

```diff
@@ -34,14 +34,31 @@ let cssClassName = if upvotes < downvotes {
   -----------------------------------------------------------------------------
   Exercise 1
   -----------------------------------------------------------------------------
-  Calculate the discount applicable to shopping cart total amount using the
+  Calculate the discount applicable to shopping cart total amount applying the
   following rules:
 
-  1. When the total is either 500 or above:
-     - a flat 10% discount.
-  2. When the total is either 200 or above, but below 500:
-    - add 25 to,
-    - a 5% discount on the amount greater than 200
+  1. When the total is either 500 or above apply a flat discount of 10%.
+
+    ```
+    // Example 1: amount = 500
+    discount = 500 * 0.1
+
+    // Example 2: amount = 520
+    discount = 520 * 0.1
+    ```
+
+  2. When the total is either 200 or above, but below 500.
```

Comment:
Same here.
"When the total is 200 or above, but below 500:
 -Apply a 5% discount on the amount greater than 200. Add 25 to this
    intermediate result to reach the final discount total. "

Reply:
Will go ahead and also break it into two separate instructions.